src/rez/tests/test_shells.py

```diff
@@ -225,19 +275,32 @@ def _print(value):
             _print(literal("hello world"))
             _print(literal("hello 'world'"))
             _print(literal('hello "world"'))
-            _print("hey %WHO%" if windows else "hey $WHO")
-            _print("hey %WHO%" if windows else "hey ${WHO}")
-            _print(expandable("%GREET% " if windows else "${GREET} ").e("%WHO%" if windows else "$WHO"))
-            _print(expandable("%GREET% " if windows else "${GREET} ").l("$WHO"))
+
+            # Generic form of variables
+            _print("hey $WHO")
+            _print("hey ${WHO}")
+            _print(expandable("${GREET} ").e("$WHO"))
+            _print(expandable("${GREET} ").l("$WHO"))
             _print(literal("${WHO}"))
-            _print(literal("${WHO}").e(" %WHO%" if windows else " $WHO"))
+            _print(literal("${WHO}").e(" $WHO"))
 
             # Make sure we are escaping &, <, >, ^ properly.
             _print('hey & world')
             _print('hey > world')
             _print('hey < world')
             _print('hey ^ world')
 
+            # Platform dependent form of variables.
```

Comment:
I would just include unix shells here also, rather than have an exception at all. It should work also and it doesn't really matter if it re-tests some of the same things.

Reply:
Removed the platform checks. Way cleaner, thanks.